Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -1214,6 +1214,7 @@ def prepGlyph(g, marker, index=0):
     if t == 'dot':
         gs.SetGlyphTypeToCircle()
         gs.FilledOn()
+        s *= numpy.pi
```

Comment:
I was looking at numbers with a scientist, 3 wasn't right and 4 too big. So I made it fun. Beside s is not an integer to start with.